packages/shared/src/components/cards/AcquisitionFormCard.tsx

```diff
@@ -4,7 +4,7 @@ import { useRouter } from 'next/router';
 import classNames from 'classnames';
 import shuffle from 'lodash.shuffle';
 import { Card } from './Card';
-import { Card as CardV1 } from './v1/Card';
+import { ListCard as CardV1 } from './list/ListCard';
```

Comment:
Why not keep the naming ListCard, it will still be confusing when going through code and finding v1

Reply:
It's just missed from the renaming cleanup :) (hate automated renames)